jpa/subsystem/src/main/java/org/jboss/as/jpa/management/DynamicManagementStatisticsResource.java

```diff
@@ -243,7 +243,7 @@ public Resource removeChild(PathElement address) {
 
     @Override
     public boolean isRuntime() {
-        return false;
+        return true;
```

Comment:
This is the main reason we started looking in to it.
This was wrongly marked as not runtime, as it clearly is, as result it was included in many operations (validations, read, ...) that shouldn't be.